requirements/edx/github.txt

```diff
@@ -63,6 +63,7 @@ git+https://github.com/mitodl/django-cas.git@47cfef16cbbf305f943cceb8c0b02514194
 git+https://github.com/edx/pyfs.git@96e1922348bfe6d99201b9512a9ed946c87b7e0b#egg=fs==0.4.0
 # The officially released version of django-debug-toolbar-mongo doesn't support DJDT 1.x. This commit does.
 git+https://github.com/hmarr/django-debug-toolbar-mongo.git@b0686a76f1ce3532088c4aee6e76b9abe61cc808#egg=django-debug-toolbar-mongo==0.1.10
+git+https://github.com/edx/django-oauth-plus.git@01ec2a161dfc3465f9d35b9211ae790177418316#egg=django-oauth-plus==2.2.8
```

Comment:
Perhaps add an explanation of why we're not using upstream here - and the conditions under which we can move back to upstream?

Reply:
And don't you need `2.2.9.edx-1` here instead?